src/components/contextual-menu-item/readme.md

```diff
@@ -0,0 +1,27 @@
+# gxg-contextual-menu-item
```

Comment:
I added all the descriptions on contextual-menu/readme.md, because that is where the story resides.
If I add comments here they wouldn´t be visible on the story.